CMakeLists.txt

```diff
@@ -54,10 +54,11 @@ set(warehouse_srcs
 add_library(warehouse_ros_mongo SHARED ${warehouse_srcs})
 target_link_libraries(warehouse_ros_mongo ${catkin_LIBRARIES} ${MongoDB_LIBRARIES} ${OPENSSL_LIBRARIES} ${Boost_LIBRARIES})
 
-if(CATKIN_ENABLE_TESTING)
-  catkin_add_gtest(test_warehouse_ros_mongo_cpp test/test_warehouse_ros_mongo.cpp)
-  target_link_libraries(test_warehouse_ros_mongo_cpp warehouse_ros_mongo)
-endif()
+# TODO: start ROS master in CI, skip tests for now
```

Comment:
The problem is the instance of the ROS master, which is needed for e.g. the time inside the test. I am not sure how this can be started in a test. Can you point me to some documentation?
I basically disabled it because the tests were disfunctional for a long time now.

Reply:
That's only the first issue. Running a roscore locally, the next issue is the missing mongodb server.
Hence my question: How do I run this one?

Reply:
I don't know. The bulk of the CI is done in this script: https://github.com/ros-planning/moveit_ci/blob/master/travis.sh. I assume The remaining services (ROS, mongoDB) need to be started in the docker container.
@davetcoleman Can you add an instance of the ROS master and the mongoDB server to the docker container?

Reply:
I misunderstood: I know about the travis.sh script. I also know how to have a ros master for tests.
I guess, you manually verified the tests on your locally machine, running a mongodb server there.
How did you start and configure your mongodb server?

Reply:
Ah yes, sorry. I installed the meta package `mongodb` on Ubuntu via apt. I think the server started automatically after this with the correct port. There was no need for special configuration. The database is actually stored locally in a folder (given by launch file argument `moveit_warehouse_database_path`).

Reply:
Which launch file you talk about? Can you provide a pointer?
mongodb should be automatically installed by rosdep inside the docker container. This shouldn't be the issue.